plugins/reporters/spdx/src/main/kotlin/Extensions.kt

```diff
@@ -35,13 +35,15 @@ import org.ossreviewtoolkit.model.ScanResult
 import org.ossreviewtoolkit.model.SourceCodeOrigin
 import org.ossreviewtoolkit.model.VcsInfo
 import org.ossreviewtoolkit.model.VcsType
+import org.ossreviewtoolkit.model.config.LicenseFilePatterns
```

Comment:
commit: Why is the result of the root license matcher only used as a fallback, and not always?
I believe it should always be used to match the spec.

Reply:
I'm considering this, please note that this is still a draft.

Reply:
The logic to populate SPDX's `licenseDeclared` also from licenses detected in main license files is now always used.